website/versioned_docs/version-0.63/cli.md

```diff
@@ -0,0 +1,77 @@
+---
+id: version-0.63-cli
+title: React Native Windows CLI
+original_id: cli
+---
+
+This guide will give you more information on the React Native Windows CLI.
+
+## `react-native-windows-init`
+
+`react-native-windows-init` is a CLI used to bootstrap the addition of the Windows platform to `react-native` projects.
+
+### Usage
+Run this from an existing `react-native` project, to install `react-native-windows` and generate initial project files for Windows.
+
+```bat
+npx react-native-windows-init --overwrite
+```
+
+### Options
+Here are the options that `react-native-windows-init` takes:
+| Option          | Input Type                                  | Description                                      |
+|-----------------|---------------------------------------------|--------------------------------------------------|
+| `--help`        | boolean                                     | Show help.                                       |
+| `--version`     | string                                      | The version of react-native-windows to use.      |
+| `--namespace`   | string                                      | The native project namespace.                    | 
+| `--verbose`     | boolean                                     | Enables logging.                                 |
+| `--language`    | string ["`cs`","`cpp`"] [default: "`cpp`"]  | Which language the app is written in.            |
+| `--projectType` | string ["`app`","`lib`"] [default: "`app`"] | The type of project to initialize.               |
+| `--overwrite`   | boolean                                     | Overwrite any existing files without prompting.  |
+| `--useWinUI3`   | boolean                                     | Targets WinUI 3.0 (Preview) instead of UWP XAML. |
```

Comment:
is this content accurate? did we have winui3-ish support back in 0.63?
I recommend not having these pages in older releases. Only update 0.65+, the rest of the releases are not officially supported anymore as of next week

Reply:
Will do! That seems reasonable to me.